democracy_club/templates/base.html

```diff
@@ -6,127 +6,130 @@
-  {% endblock twitter_tags %}
+    <link rel="alternate" type="application/atom+xml" title="Democracy Club Blog" href="{% url "hermes_post_feed" %}" />
+    <link rel="manifest" href="{% static "manifest.json" %}">
+    {% block site_icons %}
+        <link rel="shortcut icon" href="{{ CANONICAL_URL }}{% static 'images/logo_icon.png' %}">
+        <link rel="apple-touch-icon" sizes="180x180" href="{{ CANONICAL_URL }}{% static 'icons/apple-touch-icon.png' %}">
+        <link rel="icon" type="image/png" href="{{ CANONICAL_URL }}{% static 'icons/favicon-32x32.png' %}" sizes="32x32">
+        <link rel="icon" type="image/png" href="{{ CANONICAL_URL }}{% static 'icons/favicon-16x16.png' %}" sizes="16x16">
+        <link rel="mask-icon" href="{{ CANONICAL_URL }}{% static 'icons/safari-pinned-tab.svg' %}" color="#ec008c">
+    {% endblock site_icons %}
+    {% block base_og_tags %}
+        <meta name="description" property="og:description" content="{% block og_description %}We build digital tools to support everyone’s participation in UK democracy.{% endblock og_description %}" />
+        <meta name="image" property="og:image" content="{% block og_image_content %}{{ CANONICAL_URL }}{% static 'images/og_image_logo.png' %}{% endblock og_image_content %}" />
+        <meta name="title" property="og:title" content="{% block og_title_content %}{{ SITE_TITLE }}{% endblock og_title_content %}" />
+        <meta name="type" property="og:type" content="{% block og_type_content %}website{% endblock og_type_content %}">
+        <meta name="url" property="og:url" content="{% block og_url_content %}{{ CANONICAL_URL }}{{ request.path }}{% endblock og_url_content %}">
+    {% endblock base_og_tags %}
+    {% block twitter_tags %}
+        <meta name="twitter:card" content="{% block twitter_card_type %}summary{% endblock twitter_card_type %}">
+        <meta name="twitter:site" content="@democlub">
+        <meta name="twitter:image" content="{% block twitter_image %}{{ CANONICAL_URL }}{% static 'images/logo_icon.png' %}{% endblock twitter_image %}" />
+        <meta name="twitter:image:alt" content="{% block twitter_image_alt %}Democracy Club logo{% endblock twitter_image_alt %}">
+        <meta name="twitter:title" content="{% block twitter_title_content %}Democracy Club{% endblock twitter_title_content %}">
+        <meta name="twitter:url" content="{{ CANONICAL_URL }}{{ request.path }}">
+        <meta name="twitter:description" content="{% block twitter_description_content %}We build digital tools to support everyone’s participation in UK democracy.{% endblock twitter_description_content %}"/>
+    {% endblock twitter_tags %}
+    <meta name="viewport" content="width=device-width, initial-scale=1"> <!-- Ensures optimal rendering on mobile devices. -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" /> <!-- Optimal Internet Explorer compatibility -->
```

Comment:
Could you make an issue to add these (if not in already) to the dc utils base template?

Reply:
Added to issue https://github.com/DemocracyClub/dc_django_utils/issues/30

Reply:
Although I actually it's partially already there. I think I added this when I was trying to get the Paypal SDK to work; I will remove from here.